### DISCUSSION TITLE
Table _collections has no column named indexes

### DISCUSSION THREAD

**Discussion by ondrej-langr** [pocketbase/pocketbase#2512] [Q&A] [answered]
Hey! 

I have come across issue when I updated to latest version. This app runs in docker and the update was done by just updating the version number in dockerfile and pushing to remote. Nothing more has been done. 

To be more specific on how the error looks:
<img width="979" alt="image" src="https://github.com/pocketbase/pocketbase/assets/60289583/574e9866-010b-4ba3-8f1c-0b4e2f116b78">

I did not have any problem when I did this on my local machine. Please let me know if I can solve this somehow.
Thanks 😁

**Comment by ganigeorgiev** [comment]
What was the previously used version? Are you sure that your current Dockerfile is using PocketBase v0.14+?

**Reply by ganigeorgiev** [reply, marked as answer]
Based on the migration timestamp in the screenshot I guess this could happen if the user migration is after an already applied system [init migration](https://github.com/pocketbase/pocketbase/blob/master/migrations/1640988000_init.go) and before the new [add_indexes_column migration](https://github.com/pocketbase/pocketbase/blob/master/migrations/1679943781_add_indexes_column.go).
To fix it you can try first to update to v0.13.4 and after that to v0.15.3.

**Reply by ondrej-langr** [reply]
Hey! Thanks for reaching out! My previoues version vas v0.13 and after that I updated to v0.15.2. 

I think it will be useful to show you how that collection looks like in migrations: 
<img width="265" alt="image" src="https://github.com/pocketbase/pocketbase/assets/60289583/faaef6a6-4e10-4959-9a96-824608c483e2">

Anyway I will try to update to 0.13.4 first and then to latest (v0.15)

**Reply by ondrej-langr** [reply]
Nice! Lowering to v0.13.4 and to latest afterwards helped. Thanks again and have a good one 🫶
